Jenkinsfile

```diff
@@ -738,7 +738,9 @@ def rsyncResults(stage) {
     }
 }
 
-def checkLastImage() {
+def checkLastImage(stage) {
```

Comment:
Here is a snippet of how to do it.
https://paste.fedoraproject.org/paste/iAtLDDDTMg6POXx3vvvO1A
maybe we can tackle it as part of a refactoring PR if you prefer.